dotCMS/src/main/java/com/dotmarketing/portlets/personas/business/DeleteMultiTreeUsedPersonaTagJob.java

```diff
@@ -55,10 +55,8 @@ public static void triggerDeleteMultiTreeUsedPersonaTagJob(final User user,
         jobDetail.setVolatility(false);
         jobDetail.setRequestsRecovery(true);
 
-        final long startTime = System.currentTimeMillis();
-        final SimpleTrigger trigger = new SimpleTrigger("deleteMultitreeUnusedPersonasTrigger-" + randomID,
-                "delete_multitree_unusedpersonatag_triggers",
-                new Date(startTime));
+        final Trigger trigger = new TriggerBuilder().jobDetail(jobDetail)
+                .triggerGroupName("delete_multitree_unusedpersonatag_triggers").build();
```

Comment:
maybe it is better has a util for all this code

https://github.com/dotCMS/core/blob/2b0d9d698b39a8aa5247eb997fdcf4323e224930/dotCMS/src/main/java/com/dotmarketing/portlets/personas/business/DeleteMultiTreeUsedPersonaTagJob.java#L42

I don't know maybe a util method that receive the job class and job name, and the util method creted the job, trigger, etc